packages/eslint-plugin/docs/rules/no-unsafe-return.mdx

```diff
@@ -110,6 +110,14 @@ function foo2(): unknown[] {
 }
 ```
 
+## Limitations
+
+`@typescript-eslint/no-unsafe-return` only checks for `any`, `any[]`, and -only in `async` functions- `Promise<any>`.
+It will not detect `any` in type parameters for generic constructs.
```

Comment:
perhaps an (inline code snippet) example to help explain what "type parameters for generic constructs" means? I'm not exactly sure what you mean with it